solidity/contracts/converter/types/fixed-rate-pool/FixedRatePoolConverter.sol

```diff
@@ -0,0 +1,315 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "../standard-pool/StandardPoolConverter.sol";
+
+/**
+ * @dev This contract is a specialized version of the converter, which is optimized for a
+ * liquidity pool that has 2 reserves with 50%/50% weights, and a conversion-rate of 1:1.
+ */
+contract FixedRatePoolConverter is StandardPoolConverter {
+    uint128 private _rateN;
```

Comment:
any reason not to use a Fraction here?

Reply:
Changed implementation; please review.

Reply:
question is still valid, seems like Fraction makes more sense than mapping in this case

Reply:
?

Reply:
A mapping is more useful for the actual implementation. A fraction doesn't have any added value here.

Reply:
ok, can live with that